### PR TITLE
Release 10.4

### PR DESCRIPTION
**Pull request by prasanth-reddy-wavemaker-com**

### CODE REVIEW

File: learn/app-development/deployment/deployment-to-digital-ocean.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+Autoscaling is another feature that lets users scale their applications without a radical redesign of their existing systems.
+
+Hence, through this integration, we will be enabling the end-users to perform deployment operations on their DigitalOcean Kubernetes clusters.
+
+## Deployment flow
+
+Accept the DigitalOcean account token for the user. For more information, see [How to Create a Personal Access Token](https://www.digitalocean.com/docs/apis-clis/api/create-personal-access-token/). For testing, use the following token below:
+
+```
+836ae5a604051f8303efed9f46cf7a22ac369c562b7b786efa67829643ad74a5
```
Comment (prasanth-reddy-wavemaker-com) [inline learn/app-development/deployment/deployment-to-digital-ocean.md:32]: @krishnaraoch really we can share this token in the documentation?

Comment (swethakundaram) [reply learn/app-development/deployment/deployment-to-digital-ocean.md:32]: @PatnalaPriyanka is working on this documentation. It is not done, yet.

Comment (swethakundaram) [reply learn/app-development/deployment/deployment-to-digital-ocean.md:32]: We removed it.

**Comment by swethakundaram** [comment]
We are good to merge the branch **release-10.4** to **master.** 

**To do:**
Add release notes for bug fixes. Still not done. 

For any more changes, feel free to edit, or let me know.

**Pull request merged**